clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -67,6 +68,134 @@ trait GvcfTests { self: BaseIntegrationSpec =>
   }
 
+  it should "handle raw queries for specific fields only" in {
+    val sampleAlias = s"someAlias $randomId"
+    val key = GvcfKey(
+      location = Location.GCP,
+      project = "test project",
+      sampleAlias = sampleAlias,
+      version = 2
+    )
+    val gvcfPath = Some(URI.create(s"gs://path/gvcf${GvcfExtensions.GvcfExtension}"))
+    val metadata = GvcfMetadata(
+      gvcfPath = gvcfPath,
+      documentStatus = Some(DocumentStatus.Normal)
+    )
+    val expected = Map("gvcf_path" -> gvcfPath).asJson
+    val rawJsonQuery =
+      s"""{
+         |  "_source": "gvcf_path",
+         |  "query": {
+         |    "bool": {
+         |      "must": [
+         |        {
+         |          "query_string": {
+         |            "default_field": "sample_alias.exact",
+         |            "query": "\\"$sampleAlias\\""
+         |          }
+         |        }
+         |      ]
+         |    }
+         |  }
+         |}
+      """.stripMargin
+    rawQueryTest(
+      rawJsonQuery,
+      runUpsertGvcf(key, metadata),
+      _ should contain only expected
+    )
+  }
+
+  it should "handle raw queries returning more than one document" in {
+    val sampleAlias = s"someAlias $randomId"
+    val key = GvcfKey(
+      location = Location.GCP,
+      project = "test project",
+      sampleAlias = sampleAlias,
+      version = 2
+    )
+    val gvcfPath = Some(URI.create(s"gs://path/gvcf${GvcfExtensions.GvcfExtension}"))
+    val metadata = GvcfMetadata(
+      gvcfPath = gvcfPath,
+      documentStatus = Some(DocumentStatus.Normal)
+    )
+    def setup(): Future[Unit] =
+      for {
+        _ <- runUpsertGvcf(key, metadata)
+        _ <- runUpsertGvcf(
+          key.copy(project = "a different test project"),
+          metadata.copy(notes = Some("this is a different note"))
+        )
+      } yield ()
+    val expected = Map("gvcf_path" -> gvcfPath).asJson
+    val rawJsonQuery =
+      s"""{
+         |  "_source": "gvcf_path",
+         |  "query": {
+         |    "bool": {
+         |      "must": [
+         |        {
+         |          "query_string": {
+         |            "default_field": "sample_alias.exact",
+         |            "query": "\\"$sampleAlias\\""
+         |          }
+         |        }
+         |      ]
+         |    }
+         |  }
+         |}
+      """.stripMargin
+    rawQueryTest(
+      rawJsonQuery,
+      setup(),
+      _ should be(Seq(expected, expected))
+    )
+  }
+
+  it should "fail when submitting a bogus raw query that is valid json" in {
+    val rawJsonQuery =
+      s"""{
+         |  "valid": "json",
+         |  "but": "bogus",
+         |  "elasticsearch": "query"
+         |}
+      """.stripMargin
+    recoverToSucceededIf[FailedResponse] {
+      rawQueryTest(
+        rawJsonQuery,
+        Future.successful(()),
+        _ => succeed
+      )
+    }
+  }
+
+  it should "fail when submitting invalid json" in {
```

Comment:
I don't think we need this as an integration test, since it's a purely client-side failure covered by the unit tests.